lib/android-helpers.js

```diff
@@ -52,6 +52,27 @@ const UNLOCK_HELPER_PKG_ACTIVITY = ".Unlock";
 
 let helpers = {};
 
+helpers.createBaseADB = async function (opts = {}) {
+  // filter out any unwanted options sent in
```

Comment:
is this really necessary to filter them? unknown options are anyway going to be ignored